1-WebApp-OIDC/1-5-B2C/Startup.cs

```diff
@@ -56,15 +57,23 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             // Configuration to sign-in users with Azure AD B2C
-            services.AddAuthentication(AzureADB2CDefaults.AuthenticationScheme)
-                .AddAzureADB2C(options => Configuration.Bind("AzureAdB2C", options));
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                .AddSignIn("AzureAdB2C", Configuration, options => Configuration.Bind("AzureAdB2C", options));
```

Comment:
The same way to have `AddSignIn` in AAD, can be applied to B2C, leveraging `OpenIdConnectOptions`